### ISSUE TITLE
Crud insert-form not show in edit,insert mode after click on crud_insert_title

### ISSUE THREAD

**Issue by baysao** [perak/kitchen-site#58]
Hi Perak,
I try to add a field with input type is "crud".
` {
      "name": "crud",
      "title": "Crud",
      "input": "crud",
      "type": "array",
      "array_item_type": "object",
      "crud_fields": [
        {
          "name": "name",
          "title": "Name"
        }
      ],
      "crud_insert_title": "add more"
    }`
But when i try to click on "add more" button, it not show form for add more crud fields.

**Comment by perak** [comment]
Hi,

I just tested and it works for me. Do you have any errors shown in console? BTW, insert form for CRUD is modal, maybe something related to your CSS? ... can you copy & paste generated code for this form (html & css) here?

**Comment by baysao** [comment]
Hi Perak, 
I regenerate code in other computer and it's works fine.  Maybe i have problem with static caching. 
Thanks for your supports.

**Comment by baysao** [comment]
Hi Perak,
I have another question relate to change the view of crud type in dataview components.
I saw there is an option in field definition target to helper function to add logic to change display field
`"display_helper": "crud_title",`
But in dataview, 
`"helpers_code": "\"crud_title\": function(){return \"abc\"}",`
target to parent template of dataview, and display_helper not see my helper function.
How can i define helpers_code  to xxxViewTableItems template ?

**Comment by perak** [comment]
You want to use helper defined in parent template? I think it cannot be automatically generated with "kitchen"... but try to use `../` notation: try to set `"display_helper": "../crud_title"` (and please let me know does it work).

**Comment by baysao** [comment]
Dear Perak,
I try to change the display of field  in dataview component. I try to using "display_helper" on field and "helpers_code" on component. Is it correct approach and can i do it with kitchen ?

**Comment by perak** [comment]
Yes, that's correct approach.

Set `display_helper` and define your helper.

Field:
`"display_helper": "something"`

Component:
`"helpers_code": "\"something\": function() { return "xyz"; }"`

**Comment by perak** [comment]
BTW, `display_helper` is used in `dataview` and also in `form`. If you define `display_helper` at collection level, then you'l need to define it everywhere (in dataviews and forms that shows your field). Note that you can define fields inside `dataview` and `form` (not only at collection level). Also, to avoid duplication of code, maybe you can define global helper (in external file, and add that file into `copy_files` array).

Example:
```
"copy_files": [
  { "source": "external_file.js", "dest": "CLIENT_DIR/external_file.js" }
]
```

**Comment by perak** [comment]
I just added issue for global helpers: https://github.com/perak/kitchen-site/issues/59

**Comment by baysao** [comment]
Thanks Perak for help. This closes #58
